server/services/emailService.ts

```diff
@@ -48,6 +48,7 @@ export default class EmailService {
     To continue with your account registration, please confirm your email address by visiting: 
     
     ${confirmURL}`;
+    console.log(body);
```

Comment:
Do we want to log this?

Reply:
I was logging it out to test locally - easier than going through the whole signup flow.